addons/bestja_project/wizards.py

```diff
@@ -27,9 +33,9 @@ def send(self, recipients):
         self.project.check_access_rights('write')
         self.project.check_access_rule('write')
 
-        for member in recipients:
+        for recipient in recipients:
             self.project.with_context(message=self.content).send(
                 template='bestja_project.msg_project_message',
-                recipients=recipients,
+                recipients=recipient,
```

Comment:
No, recipient is an one-element recordset. This will fork fine, since the `send` method accepts any iterable.